packages/create-block/lib/init-block.js

```diff
@@ -33,7 +34,9 @@ async function initBlockJSON( {
 	info( '' );
 	info( 'Creating a "block.json" file.' );
 
-	const outputFile = join( process.cwd(), slug, folderName, 'block.json' );
+	const outputFile = ! plugin
+		? join( process.cwd(), slug, 'block.json' )
+		: join( process.cwd(), slug, folderName, 'block.json' );
```

Comment:
The code without negation in the conditional is easier to process:
```suggestion
	const outputFile = plugin
		? join( process.cwd(), slug, folderName, 'block.json' )
		: join( process.cwd(), slug, 'block.json' );
```

I see the same construct in other places. It's also probably an individual matter, so I will leave it up to you whether to leave it as is.

Reply:
Updated.